src/util/clustering.js

```diff
@@ -127,23 +127,26 @@
   };
 
   /**
-   *
    * This class manages clustering of an array of positions hierarchically.
    * The algorithm and code was adapted from the Leaflet marker cluster
    * plugin by David Leaver: https://github.com/Leaflet/Leaflet.markercluster
    *
    * @class geo.util.ClusterGroup
    * @param {object} opts An options object
+   * @param {number} width The width of the window; used for scaling.
+   * @param {number} height The height of the window; used for scaling.
```

Comment:
I'm not sure about adding the window dimensions here.  What the class needs is the `unitsPerPixel` method to determine the scaling for the clustering distance.  This is perhaps something that I can fix with the new camera API.